website/docs/01-arrays.md

```diff
@@ -7,7 +7,8 @@ next: classes.html
 ---
 
 Array types are simply instantiations of a special polymorphic Array class:
-the type `Array<T>` describes arrays whose elements are of type `T`.
+the type `Array<T>` describes arrays whose elements are of type `T`. `T[]` can
+be used as a shorthand to annotated arrays.
```

Comment:
I like the idea of mentioning this syntax here, but I think it could be worded a bit more clearly.

How about "The shorthand syntax `T[]` is equivalent to `Array<T>`."?